packages/server/graphql/mutations/addComment.ts

```diff
@@ -13,6 +14,16 @@ import AddCommentInput from '../types/AddCommentInput'
 import AddCommentPayload from '../types/AddCommentPayload'
 import validateThreadableThreadSourceId from './validateThreadableThreadSourceId'
 
+type AddCommentMutationVariables = {
+  comment: {
```

Comment:
Here I've only included the comment fields that are being used. Adding all of the possible comment fields might make the file easier to update but it would add a _lot_ of unnecessary code too